src/main/java/org/mule/extension/sftp/internal/connection/SftpClient.java

```diff
@@ -76,7 +76,7 @@ public class SftpClient {
 
   private static final Logger LOGGER = getLogger(SftpClient.class);
   protected static final OpenMode[] CREATE_MODES = {OpenMode.Write, OpenMode.Create, OpenMode.Truncate};
-  protected static final OpenMode[] APPEND_MODES = {OpenMode.Write, OpenMode.Append};
+  protected static final OpenMode[] APPEND_MODES = {OpenMode.Write, OpenMode.Append, OpenMode.Create};
```

Comment:
What is the difference between Write and Create?

Reply:
Write mode opens the existing file in write mode, if not, the create mode allows to create a new file and open it in write mode.